### DISCUSSION TITLE
Is there a way to automatically select custom layouts based on selected category of a product.

### DISCUSSION THREAD

**Discussion by dannycao-2d** [pimcore/pimcore#15490] [Q&A / Support]
I have a Product with a lot of input fields, based on the category (Many-to-one relationship) I want to be able to hide fields in the Product. I want to achieve this without having to touch the code. Is that possible?

**Comment by johanlindgren** [comment]
It is possible with some coding https://pimcore.com/docs/platform/Pimcore/Best_Practice/Showing_Custom_Layouts_based_on_Object_Data/

I'm not aware of any other method
